content/_data/changelogs/weekly.yml

```diff
@@ -4587,6 +4587,36 @@
     # pull: 4074 too minor (Use a proper example domain in hudson.FunctionsTest#testGetActionUrl_unparseable())
     # pull: 4081 too minor (Improved logging for NodeProvisioner)
     # pull: 4083 ([JENKINS-57922] - Ability to specify target directory for translation-tool.pl)
+- version: '2.183'
+  date: 2019-06-30
+  changes:
+    - type: rfe
+      pull: 4072
+      issue: 58041
+      message: |-
+        Add support for IPv6 addresses in the Jenkins URL configuration.
+    - type: rfe
+      pull: 4063
+      issue: 57959
+      message: |-
+        Update args4j from 2.0.31 to 2.33.
```

Comment:
CC @basil , but IIUC there is no real user-facing issue there in the releases. 2.32 was affected, but it impacted an unreleased Swarm Client version only. If so, I suggest removing it